### ISSUE TITLE
Why "Process.xaml" is kept in root folder? 

### ISSUE THREAD

**Issue by mrmanojrai** [UiPath/ReFrameWork#12]
The `Process.xaml` equally qualifies to be part of "Framework" flows. It has got arguments which are closely associated with other flows. Not sure why it has been kept in the root folder. 

Will it not be good to move it to be part of "Framework" folder?

**Comment by Richard5Chan** [comment]
This is just my opinion. We should not just use the technical skills but also design & usability skills so as to give user/developer a better user experience with REFramework. Putting process at the root allows user/developer to delete it, replace it or modify it with their own process(es).

**Comment by Vlek** [comment]
I agree with @mrmanojrai, this should be moved to the Framework folder. The Process.xaml file should never be deleted, the actual process logic should be added to it instead.
